lib/generators/templates/aws_ses_events/bounce_events_create.rb.erb

```diff
@@ -0,0 +1,19 @@
+class CreateSincerelyBounceEvents < ActiveRecord::Migration<%= migration_version %>
+  def change
+    create_table :sincerely_bounce_events do |t|
```

Comment:
I wonder if we should create a common events table (with STI), having the following fields:
* message_id
* delivery_system
* event_type
* recipient
* options
* timestamp